app/controllers/hosts_controller.rb

```diff
@@ -414,6 +415,34 @@ def multiple_destroy
   def multiple_build
   end
 
+  def rebuild_config
+  end
+
+  def submit_rebuild_config
+    all_fails = {:tftp => [], :dns => [], :dhcp => []}
```

Comment:
Avoid hardcoding the orchestration types into the UI, I'd suggest filling this in on the fly during the loop below.